pkgs/development/python-modules/domeneshop/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, urllib3
+, pyopenssl
+, cryptography
+, idna
+, certifi
+}:
+
+buildPythonPackage rec {
+  pname = "domeneshop";
+  version = "0.4.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "tr3wsrscIU66kTz3nlvDHj2EXoEHCH3grD0yD7BU3Fc=";
+  };
+
+  propagatedBuildInputs = [
+    urllib3
+    pyopenssl
+    cryptography
+    idna
+    certifi
+  ];
+
+  # There are none
+  doCheck = false;
+
+  pythonImportsCheck = [ "domeneshop" ];
+
+  meta = with lib; {
+    description = "Python 3 library for working with the Domeneshop API";
```

Opinionated change as Python 2 is EOL.

```suggestion
    description = "Python library for working with the Domeneshop API";
```